modules/lti/public/iframeResizer.contentWindow.min.js

```diff
@@ -0,0 +1,10 @@
+//# sourceMappingURL=iframeResizer.contentWindow.map
```

Comment:
Whats up with this line?

Reply:
That's how a minified JavaScript library often looks like… ?

Reply:
Okay, I was confused why it was commented out, but then nevermind. :D